images/win/scripts/Installers/Install-WDK.ps1

```diff
@@ -9,8 +9,8 @@ Import-Module -Name ImageHelpers -Force
 
 if (Test-IsWin19)
 {
-    $winSdkUrl = "https://go.microsoft.com/fwlink/p/?linkid=2083338"
-    $wdkUrl = "https://go.microsoft.com/fwlink/?linkid=2085767"
+    $winSdkUrl = "https://go.microsoft.com/fwlink/p/?linkid=2120843"
```

Comment:
@jpark37 is this backwards compatible with previous version that was being installed or will it break existing workflows?

Reply:
Can you be more specific? In general, having a newer SDK should still allow you to target older Windows versions.

Reply:
It is update from `10.0.18346.0` to `10.0.19041.1`. I think should be safe enough